source/prerequisite-tasks.js

```diff
@@ -71,7 +71,7 @@ module.exports = (input, pkg, options) => {
 		{
 			title: 'Check for pre-release version',
 			task: () => {
-				if (!pkg.private && version(newVersion).isPrerelease() && !options.tag) {
```

Comment:
I'm still not sure whether `isPrerelease` or `isPrereleaseOrIncrement` should be used here, as well as in `source/ui.js`.

Reply:
Here it should use `isPrerelease`, as it's checking the new normalized version, not the user input.

https://github.com/sindresorhus/np/blob/fe709ed4aa9805297118294adc12e3bfe3041d23/source/prerequisite-tasks.js#L64

Reply:
> should be used here, as well as in source/ui.js.

It's correct to use `isPrereleaseOrIncrement` there as it works on the raw user input, which might be, for example, `patch`.